src/uiTest/resources/UIScenarioSuite.conf

```diff
@@ -0,0 +1,16 @@
+probe {
```

Comment:
Pls check if there's any way to provide shared config b/w all suites (`ideprobe.conf`?)

If not supported OOTB, maybe just pass config resource name explicitly to `fixtureFromConfig(...)` somehow

Reply:
(so as to avoid repeating the same file twice)